doxia-modules/doxia-module-markdown/src/main/java/org/apache/maven/doxia/module/markdown/MarkdownParser.java

```diff
@@ -76,19 +83,23 @@ public class MarkdownParser extends AbstractTextParser implements TextMarkup {
      * In order to ensure that we have minimal risk of false positives when slurping metadata sections, the
      * first key in the metadata section must be one of these standard keys or else the entire metadata section is
      * ignored.
+     * @see <a href="https://fletcher.github.io/MultiMarkdown-5/metadata.html">Multimarkdown Metadata</a>
```

Comment:
Is this really correct? MultiMarkdown vs CommonMark?

Reply:
CommonMark does not define any metadata format.

Reply:
But FlexMark does not claim to support MultiMarkdown, does it?

Reply:
No, but Doxia Markdown supports it (since the beginning I guess). Compare also with https://github.com/vsch/flexmark-java/issues/550. I haven't touched that part, only refactored and added the underlying spec.

Reply:
See also the unmodified line 81 which mentions multimarkdown already.